crates/cxx-qt-lib/src/core/qflags/util.rs

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Joshua Booth <joshua.n.booth@gmail.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#[macro_export]
+macro_rules! impl_qflag {
```

Comment:
Please rename this macro to `unsafe_impl_qflag`, as it otherwise hides the unsafety of the trait that it implements.